docs/en/operations/system-tables/query_log.md

```diff
@@ -74,73 +80,101 @@ Columns:
     -   1 — `GET` method was used.
     -   2 — `POST` method was used.
 -   `http_user_agent` ([String](../../sql-reference/data-types/string.md)) — The `UserAgent` header passed in the HTTP request.
--   `quota_key` ([String](../../sql-reference/data-types/string.md)) — The “quota key” specified in the [quotas](../../operations/quotas.md) setting (see `keyed`).
+-   `http_referer` ([String](../../sql-reference/data-types/string.md)) — URL of the query source.
+-   `forwarded_for` ([String](../../sql-reference/data-types/string.md)) — Headers passed in the HTTP query.
+-   `quota_key` ([String](../../sql-reference/data-types/string.md)) — The `quota key` specified in the [quotas](../../operations/quotas.md) setting (see `keyed`).
 -   `revision` ([UInt32](../../sql-reference/data-types/int-uint.md)) — ClickHouse revision.
--   `thread_numbers` ([Array(UInt32)](../../sql-reference/data-types/array.md)) — Number of threads that are participating in query execution.
+-   `log_comment` ([String](../../sql-reference/data-types/string.md)) — Comment on the column, or an empty string if it is not defined.
```

Comment:
```suggestion
-   `log_comment` ([String](../../sql-reference/data-types/string.md)) — Log comment. It can be set to arbitrary string no longer than max_query_size. An empty string if it is not defined.
```